src/clue.js

```diff
@@ -1,14 +1,126 @@
 // ITERATION 1
 
+//randomize arr length to pick rnd
+// let testArr = [12,23,34]
+
+let selectRandom = (arr) => {
+
+    if (arr.length === 0) {
+        return undefined;
+    } else {
+    //the rnd item picked
+    let randomItem = arr[Math.floor(Math.random() * arr.length)]
+    return randomItem;
```

Comment:
Keep in mind that you can just return the random item as well:
return arr[Math.floor(Math.random() * arr.length)]